apps/api/lib/api/web/router.ex

```diff
@@ -2,6 +2,8 @@ defmodule API.Web.Router do
   use API.Web, :router
 
   pipeline :api do
+    plug Guardian.Plug.VerifyHeader
```

Comment:
Do we want to verify header for the create user, login and refresh token endpoints?